_rules/link-accessible-name-c487ae.md

```diff
@@ -71,72 +74,82 @@ The rule assumes that all links are [user interface components](https://www.w3.o
 
 #### Passed Example 1
 
-`a` element with [accessible name][] through content.
+This `a` element has an [accessible name][] from its content.
 
 ```html
 <a href="http://www.w3.org/WAI"> Web Accessibility Initiative (WAI) </a>
 ```
 
 #### Passed Example 2
 
-Element with [explicit role](#explicit-role) of link with [accessible name][] through content.
+This `div` element has an [explicit semantic role](#explicit-role) of `link` and an [accessible name][] from its content.
 
 ```html
-<div role="link">Web Accessibility Initiative (WAI)</div>
+<div role="link" onclick="window.location.href='https://www.w3.org/WAI/'">Web Accessibility Initiative (WAI)</div>
 ```
 
 #### Passed Example 3
 
-`Button` with an [explicit role](#explicit-role) of `link`.
+This `button` element has an [explicit semantic role](#explicit-role) of `link` and an [accessible name][] from its content.
 
 ```html
-<button role="link">Click me!</button>
+<button role="link" onclick="window.location.href='https://www.w3.org/WAI/'">Click me for WAI!</button>
 ```
 
 #### Passed Example 4
 
-[accessible name][] for link via `aria-label`
+This `a` element has an [accessible name][] via `aria-label`
 
 ```html
-<a href="http://www.w3.org/WAI"><img src="#" aria-label="This is a link"/></a>
+<a href="http://www.w3.org/WAI"
+	><img src="/test-assets/shared/w3c-logo.png" aria-label="Web Accessibility Initiative"
+/></a>
 ```
 
 #### Passed Example 5
 
-Link named via `title` on link.
+This `a` element has an [accessible name][] via `title`.
 
 ```html
-<a href="http://www.w3.org/WAI" title="This is a link"><img src="#"/></a>
+<a href="http://www.w3.org/WAI" title="Web Accessibility Initiative"
+	><img src="/test-assets/shared/w3c-logo.png" alt=""
+/></a>
 ```
 
+**Note**: There are assistive technologies that do not support using the `title` attribute for an [accessible name][], or in which this feature can be disabled.
```

Comment:
There are several of these notes in the rule. Shouldn't we be moving them to the accessibility support section?

I know it applies to this very example above, but I think the note would fit there better.

Reply:
@daniel-montalvo I've moved the notes to the accessibility support section. Can you please review again?